servers/republik/modules/crowdfundings/lib/CustomPackages/rules/earlyAdopterBonus.js

```diff
@@ -13,36 +13,36 @@ module.exports = ({ package_, packageOption, membership, payload, now }) => {
       'membership.createdAt not within early adopter range (< %s)',
       MEMBERSHIP_CREATED_BEFORE
     )
-    return
+    return true
```

Comment:
this rule always return true because it mutates and never restricts? Maybe add a comment at the top that this is a mutating rule that always passes

Reply:
Added [some more comments](https://github.com/orbiting/backends/pull/341/files#diff-925a20658e774e9b2028dac18c6ae8c2R219-R235) on rule handling ("passing", "mutating") in code parts which evaluate rules.